content/events/2024-09-12-meet-up.md

```diff
@@ -0,0 +1,20 @@
++++
+title       = "NL_CL Meetup, Eindhoven"
+description = "@Demo. This event is an open meetup of Eindhoven's live coding community."
+
+location    = "[Demo, 18 Septemberplein 17, Eindhoven NL](https://www.instagram.com/demo.eindhoven/)"
+doors       = 2024-09-12T19:30:00+01:00
+start       = 2024-09-12T20:00:00+01:00
+end         = 2024-09-12T22:00:00+01:00
+
+links = """
+  ### [» Go to eventpage ]()
```

Comment:
Hier kan je eventueel de link naar de instagram pagina invullen (misschien zodra er ook een post over gemaakt is)